tests/integration_tests/functional/test_api.py

```diff
@@ -897,7 +897,8 @@ def _drive_patch(test_microvm):
     assert test_microvm.api_session.is_status_bad_request(response.status_code)
     assert (
         "Unable to patch the block device: BackingFile(Os { code: 2, "
-        'kind: NotFound, message: \\"No such file or directory\\" })' in response.text
+        'kind: NotFound, message: \\"No such file or directory\\" }, \\"foo.bar\\")'
```

Comment:
could we use a variable for the `foo.bar`?

Reply:
fixed